config/environments/development.rb

```diff
@@ -25,7 +25,7 @@
     config.action_controller.enable_fragment_cache_logging = true
 
     config.cache_store = :mem_cache_store,
-                         { compress: true, compression_min_size: 524_288 }
+                         { compress: true, compression_min_size: 524_288, serializer: :json }
```

Comment:
failure points to this hash is missing `:json` key

https://github.com/rails/rails/blob/6f57590388ca38ed2b83bc1207a8be13a9ba2aef/activesupport/lib/active_support/cache/serializer_with_fallback.rb#L166-L172